### ISSUE TITLE
iOS 13 Memory Issue

### ISSUE THREAD

**Issue by AvdLee** [awkward/Tatsi#31]
We can see a crash in our crash reports regarding Memory, happening in the `AssetsGridViewController` and 100% on iOS 13.

We still need to look into it but I have the feeling that this is caused by loading the previews for the first time, filling up the memory.

Any ideas?

**Comment by terhechte** [comment]
@AvdLee  Did you ever find a solution for this Antoine?

**Comment by AvdLee** [comment]
in our app it turned out to be related to another retain cycle as we didn't capture the new drag down dismiss for the new iOS 13 sheet presentation style.

I'm not entirely sure whether the original description is still an issue. Do you experience the same?

**Comment by terhechte** [comment]
No, I was pondering using this library as most of the other open source solutions for multi-selection of images did not meet my requirements. However I've started using it now and so far it looks good. :)

**Comment by AvdLee** [comment]
@terhechte awesome! Yeah, we're using it in the WeTransfer app and it has since not let us down 🙂
